<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicWindow</class>
 <widget class="QMainWindow" name="MusicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/zuixiaohua.png</normaloff>:/resource/zuixiaohua.png</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/guanbi.png</normaloff>:/resource/guanbi.png</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="line">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>1</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>850</y>
      <width>1600</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="slider_musicTime">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>950</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">/*未划过区域的美化*/ 
QSlider::groove
{
	background: rgb(204, 204, 204);
	height:4px;
}
/*已划过区域的美化*/
QSlider::sub-page
{
	background-color: rgb(255,78,78);
	height:4px;
}
/*滑块的美化*/
QSlider::handle
{	
	border-image: url(:resource/play_nor);	
	width:10px;
	margin:-3px -3px -3px -3px;	
}

</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="music">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/music_logo.png</normaloff>:/resource/music_logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="friends">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>60</width>
      <height>63</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/message_logo.png</normaloff>:/resource/message_logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="main_page">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>个人主页</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/main_page_logo.png</normaloff>:/resource/main_page_logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>120</y>
      <width>271</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <widget class="QToolButton" name="btn_main_page">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>个人主页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_message">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>社交</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_music">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>我的音乐</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exit">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:nones;</string>
       </property>
       <property name="text">
        <string>退出登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="setting">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/setting_logo.png</normaloff>:/resource/setting_logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="play">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>870</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/bfzn_004.png</normaloff>:/resource/bfzn_004.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="pre_music">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>870</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/bfzn_001.png</normaloff>:/resource/bfzn_001.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="next_music">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>870</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/bfzn_002.png</normaloff>:/resource/bfzn_002.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>1150</width>
      <height>700</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="person_page">
     <widget class="QLabel" name="main_background">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1150</width>
        <height>700</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/resource/duan.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="message_page"/>
    <widget class="QWidget" name="music_page">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1131</width>
        <height>681</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">/*美化表头*/
QHeaderView::section
{
    font: 16pt &quot;楷体&quot;;
    background: transparent;
    border:none;
	padding-left:3px;
    margin-top: 5px;
    min-height:35px;
    max-height:35px;
}
/*整体对tablewidget美化*/
QTableWidget
{
    font: 14pt ;
    border: none;
    margin: 0px;
    padding: 0px;
    color: black;
	alternate-background-color: rgb(236, 65, 65);
	//background: transparent;
}
/*对item美化*/
QTableWidget::item
{
    font: 14pt ;
    color: black;
	//background: #dddddd;
}
/*鼠标悬浮于item时*/
QTableWidget::item:hover
{
    background:#dddddd;
}
/*item选中时*/
QTableWidget::item:selected
{  
    /* color: #2eb372; */
	color: rgb(236, 65, 65);
    background: #f6f6f6;

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>350</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <column>
       <property name="text">
        <string>歌曲名字</string>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>歌手</string>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>大小</string>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="btn_voice">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>940</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:nones;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/voice_open.png</normaloff>:/resource/voice_open.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="slider_volume">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>870</y>
      <width>30</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
    border-color: #cbcbcb;
}
QSlider::groove:vertical {  

background: #cbcbcb;  
width: 6px;  
border-radius: 1px;  
padding-left:-1px;  
padding-right:-1px;  
padding-top:-1px;  
padding-bottom:-1px;  
} 
QSlider::sub-page:vertical {  
background: #cbcbcb;  

border-radius: 2px;  
} 
QSlider::add-page:vertical {  
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,   
    stop:0 #439cf4, stop:1 #439cf4);  
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,  
    stop: 0 #439cf4, stop: 1 #439cf4);  
width: 10px;  
border-radius: 2px;  
}
QSlider::handle:vertical   
{     
    border-image: url(:/NiceTalk/Images/Setting/volume_nor.png);
     margin: -2px -7px -2px -7px; 
    height: 17px;
}  
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_volume">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>870</y>
      <width>35</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>30%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currDuration">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>950</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalDuration">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>950</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0:00</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton_min</zorder>
   <zorder>pushButton_close</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>slider_musicTime</zorder>
   <zorder>music</zorder>
   <zorder>friends</zorder>
   <zorder>main_page</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>setting</zorder>
   <zorder>play</zorder>
   <zorder>pre_music</zorder>
   <zorder>next_music</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>btn_voice</zorder>
   <zorder>slider_volume</zorder>
   <zorder>lb_volume</zorder>
   <zorder>label_currDuration</zorder>
   <zorder>label_totalDuration</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
